runtime/CSharp/Antlr4BuildTasks/Antlr4.CodeGenerator.targets

```diff
@@ -196,4 +196,12 @@
   </Choose>
 
   <Import Condition="'$(Antlr4IsSdkProject)' == 'True'" Project="Antlr4.CodeGenerator.DefaultItems.targets" />
+  
+  <ItemGroup Condition="'$(NCrunch)' == '1'">
+    <None Include="$(MSBuildThisFileDirectory)..\tools\**\*" />
```

Comment:
❔ Can you explain why is this part needed? Does NCrunch not work with NuGet packages?

Reply:
It will only copy the `props`/`targets` files included by NuGet (and I also think it transitively copies the imported files). It doesn't automatically copy the contents of the `tools` directory in ANTLR's case.

This is only relevant for old-style projects that use packages.config. The `PackageReference` format seems to work fine as NCrunch will reference the files from the NuGet cache directly without copying its contents, although this *may* still fail if you use a test grid (I haven't tested this combination).

Reply:
Tested again without this line: the new project format works, but the old one doesn't.

Surprisingly (at least to me), the new project format also works with a grid node - NCrunch uses its own NuGet cache in `C:\Temp\.nuget` to restore packages.

Reply:
❔ Should we condition this on `'$(Antlr4IsSdkProject)' != 'True'`?

Can you place a short comment here so future readers understand the context?